### ISSUE TITLE
bug: CLI not usable from PNPM

### ISSUE THREAD

**Issue by davidchalifoux** [triggerdotdev/trigger.dev#1311]
### Provide environment information

  System:
    OS: macOS 15.0
    CPU: (14) arm64 Apple M3 Max
    Memory: 2.70 GB / 36.00 GB
    Shell: 5.9 - /bin/zsh

  Binaries:
    Node: 20.15.1 - ~/.nvm/versions/node/v20.15.1/bin/node
    npm: 10.7.0 - ~/.nvm/versions/node/v20.15.1/bin/npm
    pnpm: 9.10.0 - /opt/homebrew/bin/pnpm
    bun: 1.1.24 - ~/.bun/bin/bun

### Describe the bug

Calling the CLI through `pnpm dlx` is not working because it detects multiple executable binaries.

```bash
> pnpm dlx trigger.dev@3.0.1 dev

 ERR_PNPM_DLX_MULTIPLE_BINS  Could not determine executable to run. trigger.dev has multiple binaries: triggerdev, trigger

Try one of the following:
pnpm --package=trigger.dev dlx triggerdev
pnpm --package=trigger.dev dlx trigger
```

When trying the recommended solution from PNPM of specifying the package, the CLI fails.

```bash
> pnpm --package=trigger.dev dlx triggerdev@3.0.1 deploy --env staging --skip-typecheck

 ENOENT  Command failed with ENOENT: triggerdev@3.0.1 deploy --env staging --skip-typecheck
spawn triggerdev@3.0.1 ENOENT

pnpm: Command failed with ENOENT: triggerdev@3.0.1 deploy --env staging --skip-typecheck
spawn triggerdev@3.0.1 ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
 ELIFECYCLE  Command failed with exit code 254.
```

### Reproduction repo

https://stackblitz.com/edit/stackblitz-starters-vyz9ci?file=package.json

### To reproduce

Check out the [Stackblitz example](https://stackblitz.com/edit/stackblitz-starters-vyz9ci?file=package.json) and try running these commands:
- `pnpm run example1`
- `pnpm run example2`

**Comment by ericallam** [comment]
Thanks for this detailed report, will get it fixed up ASAP 🫡

**Comment by ericallam** [comment]
This has been fixed in 2b5771f38c5fad8ced8253d95b3fa409dc8e6be6 and released as 3.0.2
